README.md

```diff
@@ -43,7 +43,7 @@ The recommended approach is deploying to Vercel. If you don't have an account al
 
 - Fork this repo
 - Create a new project on Vercel and connect this repo
-- It's a monorepo. So, choose either `packages/blog-starter-kit/themes/enterprise`, `packages/blog-starter-kit/themes/hashnode` or `packages/blog-starter-kit/themes/personal` as the root directory while importing on Vercel.
+- It's a monorepo, so, choose either `packages/blog-starter-kit/themes/enterprise`, `packages/blog-starter-kit/themes/hashnode` or `packages/blog-starter-kit/themes/personal` as the root directory while importing on Vercel.
```

Comment:
Can we remove the comma after `so` to create a smoother flow in the sentence.
Also, we can add a comma after `packages/blog-starter-kit/themes/hashnode`.